share/functions/abbr.fish

```diff
@@ -85,7 +85,7 @@ function abbr --description "Manage abbreviations"
 		set key $kv[1]
 		set value $kv[2]
 		# ensure the key contains at least one non-space character
-		if not string match -qr "\w" -- $key
+		if not string match -qr "[^\s]" -- $key
```

Comment:
Aren't both of these incorrect? Shouldn't the key be a contiguous sequence of non-space characters? This allows "abc def" as a valid abbreviation. But if you actually execute `abbr -a "abc def" "wtf"` as soon as you type `abc[space]` it expands to `def wtf`. It seems to me the correct regex is `'^\S+$'`.